ANALYSIS/ANALYSIS/AliAnalysisManager.cxx

```diff
@@ -1728,25 +1729,28 @@ void AliAnalysisManager::CheckBranches(Bool_t load)
 Bool_t AliAnalysisManager::CheckTasks() const
 {
    /// Check consistency of tasks.
-   #if ROOT_VERSION_CODE < ROOT_VERSION(5, 99, 0)
    Int_t ntasks = fTasks->GetEntries();
    if (!ntasks) {
       Error("CheckTasks", "No tasks connected to the manager. This may be due to forgetting to compile the task or to load their library.");
       return kFALSE;
    }
+   if (fgGetBadPtr == NULL)
+   {
+       Error("CheckTasks", "BAD PTR NOT INITIALIZED");
```

Comment:
Since this should not happen, let's make it a `Fatal()`. Also:

* Indent is three spaces in this class,
* Make the error message less shouted (`"Bad ptr not initialized"` would do)